### ISSUE TITLE
Support LWRP/URP

### ISSUE THREAD

**Issue by Helco** [microsoft/MixedReality-SpectatorView#328]
I did not find an issue regarding Leightweight Render Pipeline, for the camera scenario at least it did not work for me. I debugged this issue down to two problems:

1. `TextureManager` uses the `OnPreRender`/`OnPostRender` events by Unity which are not supported in LWRP. It seems to work when adding these lines in the `Start` method:

```csharp
            RenderPipelineManager.beginCameraRendering += (_, cam) =>
            {
                if (cam == spectatorViewCamera)
                    OnPreRender();
            };
            RenderPipelineManager.endCameraRendering += (_, cam) =>
            {
                if (cam == spectatorViewCamera)
                    OnPostRender();
            };
```

2. After that the output is mostly black because the alpha channel in the `renderTexture` is ignored in the `HoloAlpha` shader. I fixed it by modifying the blending to:

```glsl
fixed4 composite = backCol * (1 - _Alpha * frontCol.a) + frontCol * _Alpha * frontCol.a;
```

**Comment by chrisfromwork** [comment]
With these two changes were you able to get yourself unblocked? Thank you for piecing together this in depth overview of the changes that are needed.

**Comment by Helco** [comment]
Yes with these changes I could use the SpectatorView in a LWRP project. I did not test LWRP to its full extent though.

**Comment by Helco** [comment]
Quick update as #327 and its beautiful! occlusion support broke my LWRP fix, these are the additional steps to take:

- Break up `Blur.shader`  into two and `Graphics.Blit` commands per blur pass, as GrabPass isn't supported in LWRP
- In `OcclusionMask.shader` use `_CameraDepthTexture` instead of `_LastCameraDepthTexture` 
- Enable DepthTexture in the LWRP asset
- Add the LWRP Additional Camera Data component to the camera in the Compositor prefab
